assets/js/modules/analytics/datastore/accounts.js

```diff
@@ -80,12 +84,101 @@ export const actions = {
 			type: RECEIVE_ACCOUNTS_PROPERTIES_PROFILES_FAILED,
 		};
 	},
+
+	/**
+	 * Creates a new Analytics account.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param {Object} args              Argument params.
+	 * @param {string} args.accountName  Google Analytics account name.
+	 * @param {string} args.propertyName Google Analytics property name.
+	 * @param {string} args.profileName  Google Analytics profile name.
+	 * @param {string} args.timezone     Google Analytics timezone.
+	 * @return {Function} Generator function action.
+	 */
+	*createAccount( { accountName, propertyName, profileName, timezone } ) {
+		invariant( accountName, 'accountName is required.' );
+		invariant( propertyName, 'propertyName is required.' );
+		invariant( profileName, 'profileName is required.' );
+		invariant( timezone, 'timezone is required.' );
+
+		try {
+			const accountTicket = yield actions.fetchCreateAccount( { accountName, propertyName, profileName, timezone } );
+			yield {
+				type: RECEIVE_CREATE_ACCOUNT,
```

Comment:
There always needs to be a `payload` object per convention, so here we should use an empty `payload` object with the `type`.

Also this should probably be something like `CREATE_ACCOUNT_FINISHED`, since right now it's using the same name that intuitively belongs to the `receiveCreateAccount` action creator, while in the implementation here it actually doesn't - so let's use a different name.

Reply:
renamed, added empty payload.